src/games/strategy/triplea/settings/SettingInputComponentFactory.java

```diff
@@ -134,14 +136,14 @@ private static JPanel createRadioButtonPanel(
 
   }
 
-  private static <Type extends HasDefaults> SettingInputComponent build(
+  private static <T extends HasDefaults> SettingInputComponent<T> build(
```

Comment:
The thing is, that I always get confused when I see a multi letter generic Type, because I think it's a class...